README.md

```diff
@@ -235,7 +235,7 @@ JSON Schema Based Editor. JSON Editor takes a JSON Schema and uses it to generat
 - [opsdroid](https://github.com/opsdroid/opsdroid/labels/good%20first%20issue) _(label: good first issue)_ <br> An open source chat-ops bot framework.
 - [pandas](https://github.com/pandas-dev/pandas/labels/good%20first%20issue) _(label: good first issue)_ <br> Flexible and powerful data analysis / manipulation library for Python, providing labeled data structures similar to R data.frame objects, statistical functions, and much more
 - [VPN@Home](https://github.com/ezaquarii/vpn-at-home/labels/good%20first%20issue) _(label: good first issue)_ <br> 1-click OpenVPN deployment and management dashboard.
-- [SaltStack](https://github.com/saltstack/salt/labels/Low-Hanging%20Fruit) _(label: Low-Hanging Fruit)_ <br> Software to automate the management and configuration of any infrastructure or application at scale.
+- [SaltStack](https://github.com/saltstack/salt/labels/Low-Hanging%20Fruit) _(label: good first issue)_ <br> Software to automate the management and configuration of any infrastructure or application at scale.
```

Comment:
```suggestion
- [SaltStack](https://github.com/saltstack/salt/labels/good%20first%20issue) _(label: good first issue)_ <br> Software to automate the management and configuration of any infrastructure or application at scale.
```

Reply:
Hi adding the %20 is a bit confusing for me. Can you tell why it is?

Reply:
Sure!
It's ASCII encode of " " (space).
U can find more here: https://www.w3schools.com/tags/ref_urlencode.ASP